pkg/csi_driver/multishare_controller_test.go

```diff
@@ -1621,3 +1800,214 @@ func TestMultishareControllerExpandVolume(t *testing.T) {
 		})
 	}
 }
+
+func TestIsValidMaxVolSize(t *testing.T) {
+	tests := []struct {
+		val         int64
+		expectedret bool
+	}{
+		{
+			val: 0,
```

Comment:
consider add case for non-expected unit like `1024Gb` or `131072Mb`

consider add case for unsupported stop like `384 Gb`

Reply:
Good point